gatsby-config.js

```diff
@@ -92,7 +92,7 @@ module.exports = {
     {
       resolve: '@vtex/gatsby-plugin-thumbor',
       options: {
-        server: 'https://thumbor-dev-server.vtex.io',
+        server: 'https://thumbor-server.vtex.io',
```

Comment:
Yes! While looking at lighthouse logs I noticed it was complaining about the `max-age` the thumbor server was returning. I created this new thumbor server that returns `max-age=1year`. This is safe since we the images urls we use are immutable. 

This new server, however, is not behind any CDN. A future work should be made to put this server behind a CDN and use `h2` (http 2) protocol.